src/web/layout/Bulma/Bulma.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
The layout shouldn't be called `Bulma`.

It's a `Layout` :p, what if we would like to switch our styling framework ?